usr/src/uts/common/fs/zfs/txg.c

```diff
@@ -216,10 +216,11 @@ txg_sync_start(dsl_pool_t *dp)
 	mutex_exit(&tx->tx_sync_lock);
 }
 
+#define	TXG_THREAD_MAXNAMELEN	MAXCOMLEN + 1
```

Comment:
True.  This patch was ported from FreeBSD, where MAXCOMLEN is the limit (sizeof (curthread->td_name)).  However, I can use CB_MAXNAME anyway because td_name is clipped to its own limit rather than cp->c_name's.